MAUI/ImageEditor/Annotation.md

```diff
@@ -0,0 +1,106 @@
+---
```

Comment:
Add separate files 
Shape annotation
Text annotation
 **Shape annotation Topics**
1. Shape types
2. Customize shape settings
3. Delete the selected shape
4. Clear all annotations - ClearAnnotations() method
5. AnnotationSelected event
6. Add shape using ImageLoaded event.
7. Add shape with manual bounds
8. Freehand draw
9. Customize pen settings - mention applicable settings alone
 

**Text annotation topics**
1. Add text.
2. Customize text settings.
3. Delete the text - DeleteAnnotaion
4. Clear all texts - Clear annotations.
5. AnnotationSelected event
6. Add text using ImageLoaded event
7. Add text with manual bounds

Reply:
Added a above topics